GVFS/GVFS.Common/ModifiedPathsDatabase.cs

```diff
@@ -209,9 +234,14 @@ private bool TryParseRemoveLine(string line, out string key, out string error)
         }
 
         private bool ContainsParentDirectory(string modifiedPath)
+        {
+            return this.ContainsParentDirectory(modifiedPath, out _);
+        }
+
+        private bool ContainsParentDirectory(string modifiedPath, out string parentFolder)

```

Comment:
`ContainsParentDirectory` and `ContainsParentFolder` have quite similar signatures / names